src/Widgets/calendar/CalendarModel.vala

```diff
@@ -121,17 +120,19 @@ namespace DateTime.Widgets {
         /* --- Public Methods ---// */
 
         public Gee.ArrayList<Event> get_events (GLib.DateTime date) {
+            var ical_date = Util.date_time_to_ical (date);
```

Comment:
It seems like these modifications are the ones that are making it so only all day events show